src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,17 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
```

Comment:
Indeed, we could, and @MargarytaChepiga is doing a good job following the style already in that file--in another place we do it this way.